### DISCUSSION TITLE
[BUG] Add-PnPTeamsTab doesnt work with ClientID and ClientSecret

### DISCUSSION THREAD

**Discussion by 1nY0urF4c3** [pnp/powershell#1737] [General]
### Expected behavior 
A powershellrunbook in Azure (Logic-Apps) creates tabs in a Team, using clientID and clientSecret. 
![grafik](https://user-images.githubusercontent.com/92848152/158790323-1ecff45f-6f7f-4e68-914f-78a4653a7c60.png)

### Actual behavior 
Error cause of lag of permissions for Graph API 
![grafik](https://user-images.githubusercontent.com/92848152/158790530-7435601a-7e4d-4601-8abb-5ec1ae6bf50c.png)

### Steps to reproduce behavior
Set app-permissions on two ways and start the automation. 
![grafik](https://user-images.githubusercontent.com/92848152/158790799-96529f4f-56a5-43ae-91c5-fc4ff12729a3.png)
![grafik](https://user-images.githubusercontent.com/92848152/158791085-f1af1f38-f95f-41f5-a400-0c34639b9cef.png)

### What is the version of the Cmdlet module you are running?
![grafik](https://user-images.githubusercontent.com/92848152/158791336-21e41d5b-0e06-4141-a3cf-8183a1280442.png)
Add-PnPTeamsTab

I have no idea, where the lag of permissions is comming from. Some ideas? 

Best regards

**Comment by veronicageek** [comment]
@1nY0urF4c3 - On your last screenshot I can see you're using the old module (SharePointPnPPowerShellOnline) `v.3.29.2101.0`. This module is not maintained anymore.

Can you please try with the PnP PowerShell module (this repo) and try again.
https://pnp.github.io/powershell/articles/installation.html

**Comment by 1nY0urF4c3** [comment]
> @1nY0urF4c3 - On your last screenshot I can see you're using the old module (SharePointPnPPowerShellOnline) `v.3.29.2101.0`. This module is not maintained anymore.
> 
> Can you please try with the PnP PowerShell module (this repo) and try again. https://pnp.github.io/powershell/articles/installation.html

@veronicageek thank you for your feedback. I have now uploaded the current module to Azure. Attached is the screenshot. Unfortunately, a different error message now comes up, which refers to ACs.

![grafik](https://user-images.githubusercontent.com/92848152/159033094-f0933dfa-0443-4cac-83e0-64550e0cc890.png)

![grafik](https://user-images.githubusercontent.com/92848152/159033221-18f9acb4-3f35-4281-b8e9-d20f38f2c2f8.png)

**Comment by gautamdsheth** [comment]
@1nY0urF4c3 - this is by design. 
This requires you to connect PnPOnline using either a global admin account or a clientid+certificate combination.

https://pnp.github.io/powershell/articles/authentication.html#setting-up-access

or

https://pnp.github.io/powershell/articles/connecting.html#connect-using-a-clientid-and-pfx-certificate-stored-on-your-local-machine

After that you will be able to add a teams tab.

Check the docs here:

https://pnp.github.io/powershell/cmdlets/Add-PnPTeamsTab.html

The permission required for that is Group.ReadWrite.All 

Am converting this issue to a discussion since it is a configuration issue and not an issue with the library itself
